solidity/contracts/system/TBTCSystem.sol

```diff
@@ -483,10 +483,16 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
     }
 
     /// @notice Get the price of one satoshi in wei.
-    /// @dev Reverts if the price of one satoshi is 0 wei, or
-    ///      if the price of one satoshi is 1 ether.
+    /// @dev Reverts if the price of one satoshi is 0 wei, or if the price of
+    ///      one satoshi is 1 ether. Can only be called by a deposit with minted
+    ///      TDT.
```

Comment:
`Can only be called by a deposit with minted TDT` -> `Can only be called by a Deposit contract`?

Reply:
Bit torn on this one, but I think this is more accurate than the actual revert message… The revert message is more consistent and understandable, though. I'm comfortable with using the specific but slightly more complicated explanation in `@dev`.

Reply:
i'm fine with it. just thought the comment might allow for someone to infer that a deposit can possibly not have a minted TDT.

Reply:
Yeah, fair. We can always field that question if it comes up.